bfxapi/websockets/bfx_websocket.py

```diff
@@ -356,6 +358,17 @@ async def _status_handler(self, data):
         else:
             self.logger.warn('Unknown status data type: {}'.format(data))
 
+    async def _ticker_handler(self, data):
+        symbol = self.subscriptionManager.get(data[0]).symbol
+        if type(data[1]) is list:
+            raw_ticker = data[1]
+            t = None
+            if len(raw_ticker) == 10:
```

Comment:
Minor, but would be better to check the symbol suffix instead of length (explicitly `t` or `f`)